static/templates/single_message.handlebars

```diff
@@ -42,20 +42,20 @@
                     {{/if_and}}
                     <div class="message_controls{{#status_message}} sender-status-controls{{/status_message}}">
                         {{#if msg/sent_by_me}}
-                        <div class="edit_content"></div>
+                        <div class="edit_content" title="{{t 'Edit' }} (i then Enter)"></div>
```

Comment:
Same thought on "then" not being ideal here, and also maybe "edit" should have its own hotkey.